README.md

```diff
@@ -8,5 +8,14 @@ __Programming language:__ JavaScript
 __Actions:__ Read, Watch, Buy  
 __Run project:__ https://ruslanpleskun.github.io/cstd-pleskunruslan-2122/  
 __Run tests:__ `npm test`  
-__Release version:__ ![version](https://img.shields.io/badge/version-1.0-blue)  
-__Build Status:__ [![BuildAndTest](https://github.com/RuslanPleskun/cstd-pleskunruslan-2122/actions/workflows/test.yml/badge.svg?branch=feature%2Fpleskunruslan%2Fdevelop&event=push)](https://github.com/RuslanPleskun/cstd-pleskunruslan-2122/actions/workflows/test.yml)  
+__GIT TAG:__ `Streaming_Service_v2.0_21133`  
```

Comment:
tag format should be:
![image](https://user-images.githubusercontent.com/78484371/144289393-52e4748b-a1c1-489e-833b-e1a39339079a.png)

Reply:
Fixed. Could you please check again whether it's correct?